src/rules/tailwindcss.ts

```diff
@@ -0,0 +1,45 @@
+import type { TypedFlatConfigItem } from "@antfu/eslint-config";
+import { interopDefault } from "@antfu/eslint-config";
+
+export interface TailwindCssOptions {
+/**
+ * Allow only tailwind classes
+ *
+ * @default false
+ */
+  strict?: boolean;
+}
+
+/**
+ * Tailwind CSS configuration.
+ */
+export async function tailwindCss(options: TailwindCssOptions | boolean = false): Promise<TypedFlatConfigItem[]> {
+  if (options === false) {
+    return [];
+  }
+  if (options === true) {
+    options = {};
+  }
+
+  const {
```

Comment:
Nit
Here is another approach. I'm not sure which one is better.
```suggestion
  const strict = options.strict ?? false;
```

Reply:
I use this one and actually this style is from original antfu’s eslint config. I know this style is kind of controversial. I'm familiar with it, but I'll consider for other team members

Reply:
I just asked one of my team members, and they mentioned that using default values with destructuring assignment is better.

I think you can go with whichever method is more convenient for you.

It should be treated as just a nit!

Reply:
I know this is nit! but I have a lot of discuss for this syntax btw lol!